distro/adaptation/centos

```diff
@@ -131,7 +131,7 @@ libssl: openssl-devel
 libssl-dev: openssl-devel
 libstdc++6: libstdc++.x86_64
 libstdc++-7-dev: libstdc++-devel.x86_64 libstdc++-static.x86_64
-libtirpc: libtirpc1
+libtirpc1: libtirpc
```

Comment:
thanks, what is this change for?

Reply:
I did that to overcome below error while insalling tbench on RHEL 8.x/9:
```
Use: /home/LKP/lkp-tests/distro/installer/redhat install libsmbclient libtirpc1 libtirpc-devel rpcgen libsmbclient-devel libxslt popt-devel samba-devel zlib-devel
Updating Subscription Management repositories.
Last metadata expiration check: 0:12:49 ago on Tue 07 Nov 2023 11:09:31 PM CST.
Package libsmbclient-4.16.4-101.el9.x86_64 is already installed.
No match for argument: libtirpc1
Package libxslt-1.1.34-9.el9.x86_64 is already installed.
Package zlib-devel-1.2.11-34.el9.x86_64 is already installed.
Error: Unable to find a match: libtirpc1
Cannot install some packages of tbench depends
```
But now I realize this change was not necessary instead we can just remove that dependency, as I have included libtirpc-devel to take care of libirpc library dependency. I tested it out and install works fine on RHEL. Will send that change in revised push if you are fine with this way.